recipes-sota/aktualizr/aktualizr_git.bb

```diff
@@ -11,11 +11,12 @@ DEPENDS_append_class-native = "glib-2.0-native "
 
 RDEPENDS_${PN}_class-target = "lshw "
 RDEPENDS_${PN}_append_class-target = "${@bb.utils.contains('SOTA_CLIENT_FEATURES', 'serialcan', ' slcand-start', '', d)} "
-RDEPENDS_${PN}_append_class-target = "${@bb.utils.contains('SOTA_CLIENT_FEATURES', 'hsm', ' softhsm softhsm-testtoken', '', d)}"
 
 RDEPENDS_${PN}_append_class-target = " ${PN}-tools "
 RDEPENDS_${PN}-secondary_append_class-target = " ${PN}-tools "
 
+RDEPENDS_${PN}_class-target = " openssl-bin "
```

Comment:
Does aktualizr require `openssl-bin`, or just the testtoken script? I added the dependency there, but didn't realize aktualizr might need it, too.

Reply:
Sorry, that was just for debugging. Thanks!